SPECS/clamav/clamav.spec

```diff
@@ -14,7 +14,7 @@ Source0:        https://github.com/Cisco-Talos/clamav/archive/refs/tags/%{name}-
 
 # Note: Required an updated cargo cache when rust was updated to 1.72.0, added "-rev2" to the filename to indicate the new cache for this
 # specific event. Revert back to the original filename when a new cache is created for a different version.
-Source1:        %{name}-%{version}-cargo.tar.gz
+Source1:        %{name}-1.0.7-cargo.tar.gz
```

Comment:
Why was this changed? This needs to be updated by remaking the vendored cache and uploading it